docs/how-to/migrate/forkid-7-to-9.md

```diff
@@ -0,0 +1,286 @@
+    ```
+
+4. Wait for the verified batch number to catch up to the trusted batch number:
+
+    ```sh
+    export ETH_RPC_URL="$(kurtosis port print cdk-v1 zkevm-node-rpc-001 http-rpc)"
+    cast rpc zkevm_batchNumber
+    cast rpc zkevm_verifiedBatchNumber
+    ```
+
+5. When those two numbers are the same, stop the services that are going to be upgraded:
+
+    ```sh
+    kurtosis service stop cdk-v1 zkevm-executor-pless-001
+    kurtosis service stop cdk-v1 zkevm-node-aggregator-001
+    kurtosis service stop cdk-v1 zkevm-node-eth-tx-manager-001
+    kurtosis service stop cdk-v1 zkevm-node-l2-gas-pricer-001
+    kurtosis service stop cdk-v1 zkevm-node-rpc-001
+    kurtosis service stop cdk-v1 zkevm-node-rpc-pless-001
+    kurtosis service stop cdk-v1 zkevm-node-sequence-sender-001
+    kurtosis service stop cdk-v1 zkevm-node-sequencer-001
+    kurtosis service stop cdk-v1 zkevm-node-synchronizer-001
+    kurtosis service stop cdk-v1 zkevm-node-synchronizer-pless-001
+    kurtosis service stop cdk-v1 zkevm-prover-001
+    ```
+
+## Smart contract calls
+
+1. Make the required smart contract calls:
```

Comment:
Mention that this should be done in another folder (not the `kurtosis-cdk` folder).